hack/release.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+# The script will try to checkout the release version if specified.
+# This is useful for manual or older tagged version releases.
+# If a release version is not specified, it will check the latest commit
+# for a tag, if a tag has appeared on the latest commit, it will build
+# and push the binaries from the latest commit.
+RELEASE_VERSION=${1:-}
```

Comment:
nit: I think passing config would be better through environment variable instead of positional parameters.

Reply:
My intention was to keep the script usage simple, and only when in need use it manually passing an argument (positional param), which saves the user trouble to set another env var separately. Let me know if this is not ok.

Reply:
okie, no issues.